apps/shell/tash_command.c

```diff
@@ -281,8 +281,13 @@ int tash_execute_cmd(char **args, int argc)
 int tash_cmd_install(const char *str, TASH_CMD_CALLBACK cb, int thread_exec)
 {
 	int cmd_idx;
+	static int max_tashcmd = 0;
```

Comment:
Do you use max_tashcmd to print a message once? I think you can print it always. Because that is a warning case.
How about show number of exceed to know what value is approprietary?

Reply:
Yes. It was displayed only once and error is shown for all the failed commands.   It can be modified to display the number of exceed as well.